exercises/verify.cpp

```diff
@@ -9,24 +9,27 @@ The number 3 is [not] present in the array.
 #include <iostream>
 using namespace std;
 
+bool isThere(int* arr, int val, int N){
+      for(int i = 0; i < N; i++){
+           if(arr[i] == val)
+            return true; 
+      } 
+
+      return false;
+}
+
 int main()
 {
   // placeholder
-  int N[10] = {3, 4, 5, 1, 2, 3, 4, 9, 13, 0};
+  int dim = 10;
+  int N[dim] = {3, 4, 5, 1, 2, 3, 4, 9, 13, 0};
   int num;
 
   cout << "Insert a number: ";
   cin >> num;
 
-  bool found = false;
-  for(int i = 0; i < 10; i++){
-     if(N[i] == num){
-      found = true;
-      break;
-     }
-  }
-
-  cout << "The number " << num << ((found) ? " is":" is not") << " present in the array!" << endl;
+  cout << "The number " << num << ( isThere(N,num,dim) ? " is" : " is not" ) 
+       << " present in the array!" << endl;
```

Comment:
"is" è ridondante, potresti mettere `? " " : " not" e l'is lo metti a prescindere dopo `<< num <<`